docs: Robotoff (#7896)
openfoodfacts/openfoodfacts-server: docs/introduction/robotoff.md

### About
The Robotoff project is intended to complete missing information of products by prompting users to confirm predictions inferred by Artificial Intelligence algorithms. These algorithms are calculated based on "insights", which are facts about a product that have been extracted or deduced from the product pictures, ingredients, categories, labels, etc...

> ### Robotoff OpenAPI documentation
> The OpenAPI documentation of the Robotoff API is located at : https://openfoodfacts.github.io/robotoff/references/api/#tag/Questions/operation/getQuestionsRandom 
> 
> ### Robotoff documentation
> The more general Robotoff documentation is located at : 
> https://openfoodfacts.github.io/robotoff/

Reviewer: ```suggestion
### Robotoff documentation
The documentation for Robotoff is located here:

- [General Documentation][general_docs]
- [API Documentation (OpenApi)][api_docs]

[general_docs]: https://openfoodfacts.github.io/robotoff/ "General Documentation"
[api_docs]: https://openfoodfacts.github.io/robotoff/references/api/ "API Documentation"
```
